docs/source/install.rst

```diff
@@ -31,6 +31,8 @@ Download and install miniconda_. Now run the following commands:
     make create_environment
     # activate autometa conda environment
     conda activate autometa
+    # install trimap for kmer embedding
+    pip install trimap
```

Comment:
`pip install densmap-learn` should also be added.

Reply:
densmap is installed with umap-learn

Reply:
See here for more details: https://umap-learn.readthedocs.io/en/latest/densmap_demo.html#better-preserving-local-density-with-densmap